docs/how_to_dev_frontend_en.md

```diff
@@ -0,0 +1,105 @@
+# VisualDL Web App Develop
+
+VisualDL has two major components. One is the Python/C++ SDK that logs the matrix during training,
+the other is the web-app that display visualized training data.
```

Comment:
Technically, 3 components. Because the webapp is made up of two parts, the server side that serves like an API server to push content and a true "frontend" in JS

Reply:
You mean the backend server that supplies the data, right?
I kinda combine them in my mind.

Reply:
Nope, the server side (Flask) does. Frontend = server side + client side. And in our case, the SDK would be considered the "backend", but this is not the best use of the word backend, so we should just continue to use SDK for it.